crime-application-tracking/src/main/java/uk/gov/justice/laa/crime/application/tracking/config/ResourceServerConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.justice.laa.crime.application.tracking.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.oauth2.server.resource.web.BearerTokenAuthenticationEntryPoint;
+import org.springframework.security.oauth2.server.resource.web.access.BearerTokenAccessDeniedHandler;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.access.AccessDeniedHandler;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+@Order(1)
+@Configuration
+@EnableWebSecurity
+public class ResourceServerConfiguration {
+
+    public static final String SCOPE_ATS_STANDARD = "SCOPE_ats/standard";
+
+    @Bean
+    protected BearerTokenAuthenticationEntryPoint bearerTokenAuthenticationEntryPoint() {
+        BearerTokenAuthenticationEntryPoint bearerTokenAuthenticationEntryPoint = new BearerTokenAuthenticationEntryPoint();
+        bearerTokenAuthenticationEntryPoint.setRealmName("Crime Application Tracking Service API");
+        return bearerTokenAuthenticationEntryPoint;
+    }
+
+    @Bean
+    public AccessDeniedHandler bearerTokenAccessDeniedHandler() {
+        return new BearerTokenAccessDeniedHandler();
+    }
+
+    @Bean
+    SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http.sessionManagement((session) -> session.sessionCreationPolicy(SessionCreationPolicy.STATELESS))
+                .csrf((csrf) -> csrf.disable())
+                .authorizeHttpRequests(auth -> auth
+                        .requestMatchers("/open-api/**").permitAll()
+                        .requestMatchers("/actuator/**").hasAuthority(SCOPE_ATS_STANDARD)
```

Comment:
This might cause some problems for pingdom and prometheus metrics so we should open this one up. Having said that we should look at locking down access to this endpoint here (and in other services) using an IP allow list.

Reply:
Sorry, this was a hang-over from my testing (used the actuators as there are no endpoints). Updated.